PublicationsRouter/api/v3/Send.md

```diff
@@ -105,28 +105,33 @@ If you are sending only the notification JSON, the request must take the form:
 
 ### 2. Validate Metadata + Package request
 
-If you are sending binary content as well as the metadata, the request must take the form:
+If you are sending binary content as well as the metadata, the request should be formed using [RFC 2387](https://www.ietf.org/rfc/rfc2387.txt):
 
     POST /validate?api_key=<api_key>
     Header:
-        Content-Type: multipart/form-data; boundary=FulltextBoundary
+        Content-Type: multipart/related; boundary=FullTextBoundary
```

Comment:
Can you revert to "FulltextBoundary" as publishers currently looking at using our API may already be designing /building using that value, and I'm not aware of a compelling reason to change it.

Reply:
this is less an example of an actual request and more a generic exmaple. This isn't going to work.

I just wanted to put *something* here.

Reply:
I just don't really know what ot write. The curl request itself isn't enough, but the body is never going to be correct.